articles/npm-packages/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/npm-packages/
+title: Create and Deploy NPM Packages
+description: Introduction to node modules and a hands-on about how to create and deploy a node module to the npm registry.
+author: saiharsha-balasubramaniam
+date: 2020-09-23T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/npm-packages/hero.jpg
+    alt: NPM
+---
+
+Node.js is a popular server-side language that is used by millions of developers worldwide. While building Node.js projects, we use various packages that make development easier. All of these packages are managed by the Node Package Manager.
```

Comment:
You could link to a Section article that explains what Node.js is. I see you've done so below but it would be great here as well.